charts/ceph-csi-cephfs/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 apiVersion: v1
-appVersion: v3.4.0
+appVersion: v3.4-canary
 description: "Container Storage Interface (CSI) driver,
```

Comment:
`version: 3.4.0-canary` should be `version: 3.4-canary`. its correct for rbd but for cephfs its not updated

Reply:
ah... correcting it. Thanks!

Reply:
@Madhu-1 done.. thanks